plone/transformchain/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
Just FYI but this is actually not needed, if the file is completely empty it is fine to leave it like that :-) at least flake8 will not complain